debian/zookeeper/Dockerfile

```diff
@@ -25,9 +25,12 @@ LABEL io.confluent.docker.build.number=$BUILD_NUMBER
 MAINTAINER partner-support@confluent.io
 LABEL io.confluent.docker=true
 
+ARG RELEASE_QUALITY=${RELEASE_QUALITY}
+
 ENV COMPONENT=zookeeper
 
 RUN echo "===> installing ${COMPONENT}..." \
+    && if [ "${RELEASE_QUALITY}" = "beta" ] || [ "${RELEASE_QUALITY}" = "rc" ]; then CONFLUENT_PLATFORM_LABEL=""; fi \
```

Comment:
While we don't build ZK, it is installing from kafka package below, therefore the same rules apply as my other comment.